autoendpoint/src/settings.rs

```diff
@@ -169,9 +169,11 @@ impl Settings {
     // public key, but that may not always be true.
     pub fn tracking_keys(&self) -> Vec<String> {
         let keys = &self.tracking_keys.replace(['"', ' '], "");
-        Self::read_list_from_str(keys, "Invalid AUTOEND_TRACKING_KEYS")
-            .map(|v| v.to_owned())
-            .collect()
+        let result = Self::read_list_from_str(keys, "Invalid AUTOEND_TRACKING_KEYS")
+            .map(|v| v.to_owned().replace("=", ""))
+            .collect();
+        trace!("🔍 tracking_keys: {:?}", result);
```

Comment:
```suggestion
        trace!("🔍 tracking_keys: {result:?}");
```

Reply:
Forgot this one FYI (but I also don't care)

Reply:
Meh, if i am fixing the other thing...